src/index.ts

```diff
@@ -4,3 +4,5 @@ import PostgrestQueryBuilder from './lib/PostgrestQueryBuilder'
 import { PostgrestBuilder } from './lib/types'
 
 export { PostgrestClient, PostgrestFilterBuilder, PostgrestQueryBuilder, PostgrestBuilder }
+
+export * from './lib/types'
```

Comment:
Exports all types for developers to use.
Required for next PR on `supabase-js`.